### PR TITLE
Fix Markdown links in SUPPORT.md

### PR DESCRIPTION
**Pull request by Notoh**
Add link to CONTRIBUTING.md and fix (presumably broken?) link to Github issues in SUPPORT.md

CLA: trivial

##### Checklist

- [x] documentation is added or updated

### CODE REVIEW

File: SUPPORT.md [hunk 1]
```diff
@@ -37,7 +37,7 @@ consider searching the mail archives and posting a question to the
 
 ### Open an Issue
 
-If you wish to report a bug, please open an [issue][github-issues] on GitHub
+If you wish to report a bug, please open an [issue](../../issues/new/choose) on GitHub
```
Comment (kroeckx) [inline SUPPORT.md:40]: I think you should make a link at the end of the document in the links section, like the other links in the document.

Comment (Notoh) [reply SUPPORT.md]: Done, that does indeed make much more sense.

**Review by t8m — APPROVED** [review]
I am OK with CLA: trivial.
Applicable to both master and 3.0 branches.

**Review by paulidale — APPROVED** [review]
Agreed, trivial

**Comment by paulidale** [comment]
Merged to master and 3.0, thanks for the fix.

**Pull request closed without merging**